### ISSUE TITLE
Use brand as <title>

### ISSUE THREAD

**Issue by caugner** [osmontrouge/caresteouvert#262]
**Is your feature request related to a problem? Please describe.**
* Currently the document title (`<title>`) changes according to the used locale.
* This is confusing some of our users.

**Describe the solution you'd like**
Could we always use the brand (from the `config.json`) as the `<title>` to be consistent?

**Describe alternatives you've considered**
None.

**Additional context**
Background: https://github.com/bleibtoffen/caresteouvert-de/issues/45

**Comment by PanierAvide** [comment]
@francois2metz could this issue be solved by pre-rendering some of the HTML markup on server-side ? It's quite similar to the issue with SEO image we discussed yesterday.

**Comment by francois2metz** [comment]
I don't understand the issue as the `config.json` brand depends of the language.

**Comment by pyrog** [comment]
Or is it ?
if there is no tag `name` then display tag `brand`

**Comment by PanierAvide** [comment]
@pyrog I think by brand they mean here the website name "ça reste ouvert" or "bleibtoffen" in german.

@francois2metz the issue is that in index.html, we don't use config.json title value as now.

**Comment by pyrog** [comment]
Ok, thanks 😄

**Comment by caugner** [comment]
It looks like my original concern was fixed, i.e. the `<title>` now depends on the `config.json` brand and no longer comes from the locale.

However, when you open https://www.bleibtoffen.de/, the tab title will shortly show "Ca reste ouvert", before it switches to "Bleibt offen". Then again, solving #265 (e.g. by pre-rendering) would probably also solve this issue.

**Comment by francois2metz** [comment]
Unfortunatly, pre-rendering doesn't solve that.

**Comment by caugner** [comment]
> Unfortunatly, pre-rendering doesn't solve that.

If the response would already contain the correct title, why not?

**Expected**:
```terminal
$ curl -s https://www.bleibtoffen.de/
<!DOCTYPE html><html lang="fr"><head><meta charset="utf-8"><title>Bleibt offen | Die kollaborative Karte der Orte, die weiterhin geöffnet sind</title>...
```

**Actual**:
```terminal
$ curl -s https://www.bleibtoffen.de/
<!DOCTYPE html><html lang="fr"><head><meta charset="utf-8"><title>Ça reste ouvert | La carte collaborative des lieux qui restent ouverts durant le confinement</title>...
```

**Comment by caugner** [comment]
What web servers are involved in returning that HTML? Nginx? Or is that all handled by Scalingo?

**Comment by pyrog** [comment]
You could setup your website with 😉 
* index.fr.html
* index.de.html
* index.en.html

And if you could see that keywords are not translated 😉

**Comment by pyrog** [comment]
> What web servers are involved in returning that HTML?

NGINX https://en.wikipedia.org/wiki/OpenResty

Source:
```
status: 200
Content-Type: text/html; charset=UTF-8
Last-Modified: Thu, 09 Apr 2020 15:34:06 GMT
Date: Thu, 09 Apr 2020 16:29:03 GMT
Content-Encoding: gzip
Cache-Control: public, max-age=0
ETag: W/"503-1715f93adb0"
Server: openresty
x-powered-by: Express
Strict-Transport-Security: max-age=31536000
x-request-id: 83e9f943-309b-42b6-a32b-608f995ac579

```

**Comment by caugner** [comment]
I don't know OpenResty, but since it's Lua, maybe it's possible to create a conditional replacement of the text between `<title>...</title>`: https://github.com/openresty/replace-filter-nginx-module#replace_filter

**Comment by pyrog** [comment]
http://www.guido-flohr.net/simple-content-negotiation-for-nginx/

I made a mistake:
```shell
$ ls htdocs/path/to/directory
index.html.fr
index.html.en
index.html.de
index.html.de-CH
index.html.de-AT

```

**Comment by PanierAvide** [comment]
Related to #301

**Comment by francois2metz** [comment]
Fixed by #301. Still some workd to do to configure all domains and split language and country.
